apps/landing-page/pages/blog/_meta.json

```
@@ -9,6 +9,7 @@
       "typesetting": "article"
     }
   },
+  "topology-plugin-coming-soon": "Topology plugin coming soon to a Backstage near you!",
```

```suggestion
  "topology-plugin-coming-soon-to-a-backstage-near-you": "Topology plugin coming soon to a Backstage near you!",
```

You will also need to rename the mdx file and image directory name.